server/views/pages/activities/waitlist-application/status.njk

```diff
@@ -38,22 +38,22 @@
                     errorMessage: validationErrors | findError('status'),
                     items: [
                         {
-                            value: StatusEnum.APPROVED,
-                            html: waitlistStatusBadgeWithHint(StatusEnum.APPROVED, 'Add the applicant to the waitlist. They’re ready to be allocated.'),
-                            checked: session.waitListApplicationJourney.status == StatusEnum.APPROVED
+                            value: WaitingListStatusOptions.APPROVED,
+                            html: waitlistStatusBadgeWithHint(WaitingListStatusOptions.APPROVED, 'Add the applicant to the waitlist. They’re ready to be allocated.'),
+                            checked: session.waitListApplicationJourney.status == WaitingListStatusOptions.APPROVED
                         },
                         {
-                            value: StatusEnum.DECLINED,
-                            html: waitlistStatusBadgeWithHint(StatusEnum.DECLINED, 'Reject the application and inform the person concerned.'),
-                            checked: session.waitListApplicationJourney.status == StatusEnum.DECLINED
+                            value: WaitingListStatusOptions.DECLINED,
+                            html: waitlistStatusBadgeWithHint(WaitingListStatusOptions.DECLINED, 'Reject the application and inform the person concerned.'),
```

Comment:
I can see this said 'Reject' in the description before.  I thought we were supposed to use 'Decline' instead of reject for applications.

Reply:
Checked this with Kieron. He's happy with the current description.

Reply:
Fair enough but seems odd to me given we use 'decline' everywhere.